src/java/org/apache/cassandra/metrics/DecayingEstimatedHistogramReservoir.java

```diff
@@ -91,7 +92,7 @@ public class DecayingEstimatedHistogramReservoir implements SnapshottingReservoi
      */
     public static final int DEFAULT_BUCKET_COUNT = 164;
     public static final int LOW_BUCKET_COUNT = 127;
-    public static final int DEFAULT_STRIPE_COUNT = Integer.parseInt(System.getProperty("cassandra.dehr_stripe_count", "2"));
+    public static final int DEFAULT_STRIPE_COUNT = DEHR_STRIPE_COUNT.getInt();
```

Comment:
I woud rename this to `DECAYING_ESTIMATED_HISTOGRAM_RESERVOIR_STRIPE_COUNT` (not property name, the name of enum).

Reply:
Fixed.